### PR TITLE
adding auto-build in circleci

### PR DESCRIPTION
**Pull request by DannyDouglass**
@apechimp first shot at auto-building new repos in circleci w/ their non-documented api support for following projects (kicks off the first build).

once this is working i will move on to auto-adding the heroku deploy user and perhaps the new heroku app!

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -66,8 +79,34 @@ export default createServer((req, res) => {
               if (teamPermRes.statusCode !== 204) {
                 return errorHandler(`${teamPermRes.statusCode} from github change perms`)
               }
-              res.writeHead(200, { 'Content-Type': 'text/plain' })
-              res.end(`Created ${name} at ${url} for ya!`)
+
+              if (!circleCiToken) {
```
Comment (DannyDouglass) [inline server.js]: i'd like to test this condition, have to look at handling a different setup to be able to test this though (omitting the `circleCiToken`)

Comment (apechimp) [inline server.js]: TBH I don't think we even need this check. The slack token one is just for authenticating the slash command came from us.

**Comment by apechimp** [comment]
:sparkles: :sparkles::sparkles:
:sparkles::shipit::sparkles:
:sparkles::sparkles:sparkles:

File: server.js (hunk 1 above)
Comment (apechimp) [inline server.js]: If you want to test it though, you can use [require-uncached](https://github.com/sindresorhus/require-uncached).

**Comment by apechimp** [comment]
Also, keep in mind most of our repos don't deploy to heroku :wink:

File: server.js (hunk 1 above)
Comment (DannyDouglass) [inline server.js]: it's really a placeholder for plans i have to upgrade `slack-commander` and provide more structure for input coming from slack

**Comment by DannyDouglass** [comment]
yep, most don't, but the ones that do can be automated!

**Pull request merged**